Source/Utils/WarpXMovingWindow.cpp

```diff
@@ -216,7 +220,17 @@ WarpX::shiftMF (MultiFab& mf, const Geometry& geom, int num_shift, int dir,
 
     MultiFab tmpmf(ba, dm, nc, ng);
     MultiFab::Copy(tmpmf, mf, 0, 0, nc, ng);
-    tmpmf.FillBoundary(geom.periodicity());
+
+    // Not sure why this is needed, but it is...
```

Comment:
Could you add more details on why this is needed here?

Reply:
I tested it again, and I think it is not... I initialized a `TheZeroVector` instead.